src/app/kbv/steps.js

```diff
@@ -28,16 +28,30 @@ module.exports = {
     next: "load-question",
   },
   "/prove-identity-another-way": {
-    resetJourney: true,
+    backLink: null,
     entryPoint: true,
+    fields: ["abandonRadio"],
     controller: proveIdentityAnotherWayController,
-    prereqs: ["/kbv/load-question"],
     next: [
       {
+        field: "abandonRadio",
+        value: "stop",
+        next: "/oauth2/callback?error=access_denied",
```

Comment:
Could we keep this as the default `next` step as before?
and shouldn't it be just `oauth2/callback` ?